.github/workflows/token-dashboard.yml

```diff
@@ -0,0 +1,87 @@
+name: Token Dashboard dApp
```

Comment:
```suggestion
name: Token Dashboard
```

Reply:
To better differentiate between this workflow and the one that pushes Mainnet preview, the file names were changed to `dashboard-testnet.yml` and `dashboard-mainnet.yml`. And I also changed the names of the workflows to `Token Dashboard \ Testnet` and Token Dashboard \ Mainnet`. See cc798e68cfb348b114068f859bfc8bb6e41762f7 and 518caea53737b1616fa46ebca92babc2db52e211.